src/TDBMService.php

```diff
@@ -26,7 +26,7 @@
 use Doctrine\Common\Cache\ClearableCache;
 use Doctrine\Common\Cache\VoidCache;
 use Doctrine\DBAL\Connection;
-use Doctrine\DBAL\DBALException;
+use Doctrine\DBAL\Exception;
```

Comment:
Maybe prefer an alias to show that it comes from DBAL:
```suggestion
use Doctrine\DBAL\Exception as DBALException;
```